README.md

```diff
@@ -47,11 +59,13 @@ sudo pip3 install .
 # Install with nice cli features
 sudo pip3 install ".[niceCli]"
 ```
-#### Example usage
+
+## Example usage
+
 ```bash
 arf-to-graph scan-data/ssg-fedora-ds-arf.xml xccdf_org.ssgproject.content_rule_audit_rules_unsuccessful_file_modification_creat
```

Comment:
I would just tell the user where to get `ssg-fedora-ds-arf.xml` - that it is in your repository as test data, and that it's a standardised document or output of OpenSCAP.